k8s/gitlab/release.yaml

```diff
@@ -46,6 +46,18 @@ spec:
 
       initialRootPassword: {}
 
+      email:
+        from: admin@gitlab.spack.io
+        reply_to: noreply@gitlab.spack.io
+      smtp:
+        enabled: true
+        address: email-smtp.us-east-1.amazonaws.com
+        user_name: AKIAYSCIUVA2HA4WTV6O
```

Comment:
This looks like a credential just begging to be rotated.  Is it?

Reply:
I believe that is the IAM identity as opposed to the IAM secret.

Reply:
Good catch, I hate it too. [the GitLab helm chart](https://docs.gitlab.com/charts/installation/command-line-options.html#outgoing-email-configuration) supports reading the password from a secret but not the username for some reason.

So far no rotation is set up for these credentials. It's perhaps worth pointing out that AWS SES SMTP credentials != AWS IAM credentials.

Reply:
But maybe it's a good idea to take both this and the secret from a k8s secret, so that we don't have to update this config when the secret changes.  This will need to change with the secret.

Reply:
looks like I overlapped with Zack.  It think in this case since this is just for SMTP sending, we can skip rotation for now.

Reply:
It is annoying that the configuration doesn't make this particularly easy.